### PR TITLE
Fix fabric connector issues

### PR DESCRIPTION
**Pull request by Ram-srini**
1. Change worker sync logic (as part of git issue https://github.com/hyperledger/avalon/issues/321)
2. Removed the worker loookup filtering based on worker status

Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: blockchain_connector/avalon_blockchain_connector/fabric/fabric_connector.py [hunk 1]
```diff
@@ -218,3 +163,110 @@ def workorder_event_handler_func(self, event, block_num, txn_id, status):
                 logging.info("work_order_get_result is failed")
         else:
             logging.info("work_order_submit is failed")
+
+    def _lookup_workers_in_kv_storage(self):
+        """
+        Retrieves the worker ids from shared kv using
+        worker_lookup direct API.
+        Returns list of worker id
```
Comment (danintel) [inline blockchain_connector/avalon_blockchain_connector/fabric/fabric_connector.py]: s/id/ids/

File: blockchain_connector/avalon_blockchain_connector/fabric/fabric_connector.py [hunk 2]
```diff
@@ -218,3 +163,110 @@ def workorder_event_handler_func(self, event, block_num, txn_id, status):
...
+                              + " blockchain")
+
+        for wid in wids_onchain:
+            # Mark all stale workers on blockchain as decommissioned
+            if wid not in wids_kv:
+                worker = self.__fabric_worker.worker_retrieve(wid)
+                # worker_retrieve returns tuple and first element
+                # denotes status of worker.
+                worker_status_onchain = worker[0]
+                # If worker is not already decommissioned, mark it decommission
```
Comment (danintel) [inline blockchain_connector/avalon_blockchain_connector/fabric/fabric_connector.py]: s/decommission/decommissioned/

File: blockchain_connector/avalon_blockchain_connector/fabric/fabric_connector.py [hunk 3]
```diff
@@ -218,3 +163,110 @@ def workorder_event_handler_func(self, event, block_num, txn_id, status):
...
+            return ""
+        else:
+            return worker_info["result"]
+
+    def _lookup_workers_onchain(self):
+        """
+        Lookup all workers on chain to sync up with kv storage
+        Return list of worker ids
+        """
+        worker_lookup_result = self.__fabric_worker.worker_lookup(
```
Comment (manojgop) [inline blockchain_connector/avalon_blockchain_connector/fabric/fabric_connector.py]: In a later PR we should try to create an common abstract class which can take blockchain and direct instance as argument to the function as the logic remains same for fabric/ethereum blockchain

File: sdk/avalon_sdk/fabric/fabric_work_order.py [hunk 4]
```diff
@@ -104,9 +104,35 @@ def work_order_get_result(self, work_order_id, id=None):
         # take some time to commit to chain.
         # Instead of calling contract api to get result chosen the
         # event based approach.
+        def handle_fabric_event(event, block_num, txn_id, status):
```
Comment (manojgop) [inline sdk/avalon_sdk/fabric/fabric_work_order.py]: In a later PR , we can use a common approach for event handling with asyncio construct for both fabric and ethereum

**Pull request merged**